bigtop-tests/smoke-tests/logstash/TestLogstashSmoke.groovy

```diff
@@ -0,0 +1,71 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.junit.BeforeClass
+import org.junit.AfterClass
+import org.apache.bigtop.itest.shell.Shell
+import static org.junit.Assert.assertTrue
+import static org.junit.Assert.assertFalse
+import org.junit.Test
+import org.apache.bigtop.itest.TestUtils
+import org.junit.runner.RunWith
+import static org.junit.Assert.assertEquals
+
+class TestLogstashSmoke {
+  static Shell sh = new Shell("/bin/bash -s");
+
+  static final String LOGSTASH_CMD = "/usr/lib/logstash/bin/logstash "
+  static final String TINYBENCH_CONFIG = "resources/generator.conf "
+  static final String FILEPIPELINE_CONFIG = "resources/filepipeline.conf "
+
+  @AfterClass
+  public static void LogstashCleanUp() {
+    sh.exec("rm -f resources/output.log");
+    assertTrue("Logstash cleanup failed. ", sh.getRet() == 0);
+  }
+
+  @Test
+  public void LogstashTinyBenchTest() {
+    /* Tiny benchmark with logstash Generator-plugin */
+    sh.exec("time "
+      + LOGSTASH_CMD
+      + "-f "
+      + TINYBENCH_CONFIG
+    );
+    def out = sh.out.join('\n');
+    assertFalse("Logstash tiny benchmark failed", (out =~ /ERROR/).find());
+  }
+
+  @Test
+  public void LogstashFilePipelineTest() {
+    def FILE_EXPECTED = 'resources/output-expected.log';
+    def FILE_OUTPUT = 'resources/output.log';
+
+    sh.exec(LOGSTASH_CMD
+      + "-f "
+      + FILEPIPELINE_CONFIG
+      + "&"
+    );
+
+    // Wait for completion
+    sleep(50000);
```

Comment:
I'm a bit worry about this one. So for the test to be finished we need to wait for 50 seconds?

Reply:
This `FILEPIPELINE ` operation would be performed in the background and output a result file.
It takes time ranging from 20~40 seconds on different platform hosts (x86/Arm64) from different vendors.  
Maybe there are other synchronization mechanisms for this operation but i'm sorry i don't know. 50 seconds may be a safe estimate to wait for the operation to complete which is performed asynchronously.

Reply:
https://www.elastic.co/guide/en/logstash/current/plugins-outputs-file.html#plugins-outputs-file-flush_interval
 Try set it to 0 so that it writes more frequently. I'm not an expert though...

Reply:
That's informative, thanks!
The `flush_interval => 0` makes the waiting interval reduce to 5 seconds on x86.
I will update this PR after verifying it on Arm64.